packages/bridge-ui/src/domain/chain.ts

```diff
@@ -137,3 +137,8 @@ export const taiko: WagmiChain = {
   //   address: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e",
   // },
 };
+
+// Will help us to map from chain id to RPC provider
+export const providers = new Map<number, ethers.providers.JsonRpcProvider>();
```

Comment:
do we need to use a map? maybe we can use a regular object / dictionary. if we use a regular object with type inference, we can enforce the specific chainId is only accessible from the dictionary.

Reply:
No, we don't, but this is used in some models that take a Map, and I didn't want to refactor these models

Reply:
Using the dictionary would require us to make hardcoded changes to the object type unless we inject the expected chainIDs via env var and use them in the typing. IE: not all of our taiko chains are `167001`, so enforcing that as a rule means a code PR to allow `167002` or otherwise.